script/translations/error.py

```diff
@@ -8,3 +9,27 @@ def __init__(self, reason, exit_code=1):
         """Initialize the exit app exception."""
         self.reason = reason
         self.exit_code = exit_code
+
+
+class JSONDecodeErrorWithPath(json.JSONDecodeError):
+    """Subclass of JSONDecodeError with additional properties.
+
+    Additional properties:
+      path: Path to the JSON document being parsed
+    """
+
+    def __init__(self, msg, doc, pos, path):
+        """Initialize."""
+        lineno = doc.count("\n", 0, pos) + 1
+        colno = pos - doc.rfind("\n", 0, pos)
+        errmsg = f"{msg}: file: {path} line {lineno} column {colno} (char {pos})"
+        ValueError.__init__(self, errmsg)
+        self.msg = msg
+        self.doc = doc
+        self.pos = pos
+        self.lineno = lineno
+        self.colno = colno
+
+    def __reduce__(self):
+        """Reduce."""
+        return self.__class__, (self.msg, self.doc, self.pos)
```

Comment:
```suggestion
        return self.__class__, (self.msg, self.doc, self.pos, self._path)
```

If we don't want to extend the `__reduce__` function, then we should removed it as currently it is the same as in the parent.

Reply:
Hmm, right.
Do you think we should extend it, adding self.path? How will it be used?

Reply:
I'm not familiar with pickle and the [`__reduce__` function](https://docs.python.org/3/library/pickle.html#object.__reduce__) but I think we should adopt it as the `path` is required to create the error